src/libraries/Microsoft.PowerFx.Connectors/Internal/ConnectorGlobalContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    internal class ConnectorGlobalContext
+    {
+        /// <summary>
+        /// List of functions in the same swagger file. Used for resolving dynamic schema/property.
+        /// </summary>
+        internal IReadOnlyList<ConnectorFunction> FunctionList { get; }
+
+        /// <summary>
+        /// Dictionary of connector global values like 'connectionId' or predetermined parameters.
+        /// </summary>
+        internal IReadOnlyDictionary<string, FormulaValue> ConnectorValues { get; }
```

Comment:
>ConnectorValues [](http://example.com/codeflow?start=59&length=15)

List of fixed parameters that are defined in a function, like connection ID, server name, database name...